### DISCUSSION TITLE
I want to delete it in cube form

### DISCUSSION THREAD

**Discussion by inz125** [neperfepx/neper#603] [Q&A] [answered]
Hi.

I want to remove cubes from a cube to create the model I want to use.
However, trying to remove it using -transform does not work.

![Image](https://user-images.githubusercontent.com/112613551/200780799-81357e72-221b-4a8f-b4f7-a38ca279519e.jpeg)

neper -T -n 1000 -domain "cube(10,10,10)" -transform "cut(cubei(6,11,6,11,-1,11))" -o cut1-1

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.4.2-34
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 4 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : Loading initialization file `/home/inuzuka/.neperrc'...
Info   : ---------------------------------------------------------------
Info   : MODULE  -T loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] -n 1000 -domain cube(10,10,10) -transform
         cut(cubei(6,11,6,11,-1,11)) -o cut1-1
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   : Creating domain...
Info   : Creating tessellation...
Info   :   - Setting seeds...
Info   :   - Generating crystal orientations...
Info   :   - Running tessellation...
Info   : Transforming tessellation...
Info   :   - Cutting (experimental)...
Info   :     > hspace(-6,-1,0,0)...
Info   :     > hspace(11,1,0,0)...
Info   :     > hspace(-6,0,-1,0)...
Info   :     > hspace(11,0,1,0)...
Info   :     > hspace(1,0,0,-1)...
Info   :     > hspace(11,0,0,1)...
Segmentation fault

Please help me.

**Comment by rquey** [comment, marked as answer]
This domain geometry cannot be generated at the moment...
